src/main/webapp/resources/js/pages/projects/linelist/components/LineList/LineListLayoutComponent.jsx

```diff
@@ -4,6 +4,7 @@ import { Table } from "../Table";
 import { Toolbar } from "../Toolbar";
```

Comment:
File comment

Reply:
Updated in 96bded4335584611075caa65245bc97e17888637